datasets/hr/public_officials/crawler.py

```diff
@@ -0,0 +1,126 @@
+import csv
+from datetime import datetime
+from typing import Any, Optional
+
+from pantomime.types import CSV
+
+from zavod import Context, Entity
+from zavod import helpers as h
+from zavod.logic.pep import categorise
+
+FIELD_NAMES = [
+    "Ime",
+    "Prezime",
+    "Primarna dužnost",  # first affiliation
+    "Primarna Pravna osoba u kojoj obnaša dužnost",
+    "Primarna Datum početka obnašanja dužnosti",
+    "Primarna Datum kraja obnašanja dužnosti",
+    "Sekundarna dužnost",  # second affiliation
+    "Sekundarna Pravna osoba u kojoj obnaša dužnost",
+    "Sekundarna Datum početka obnašanja dužnosti",
+    "Sekundarna Datum kraja obnašanja dužnosti",
+]
+
+
+def convert_date(date: Optional[str]) -> Optional[str]:
+    return datetime.strptime(date, "%d/%m/%Y").strftime("%Y-%m-%d") if date else None
```

Comment:
Could you use our https://zavod.opensanctions.org/helpers/#zavod.helpers.parse_date helper instead? Lots of examples in the code